app/_src/using-mesh/managing-ingress-traffic/gateway-api.md

```diff
@@ -0,0 +1,345 @@
+demo-app-5845d6447b-v7npw     2/2     Running   0          2m8s
+kuma-4j6wr-58998b5576-25wl6   1/1     Running   0          30s
+
+kubectl get svc -n kuma-demo
+NAME         TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+redis        ClusterIP      10.43.223.223   <none>        6379/TCP         3m27s
+demo-app     ClusterIP      10.43.216.203   <none>        5000/TCP         3m27s
+kuma-pfh4s   LoadBalancer   10.43.122.93    172.20.0.3    8080:30627/TCP   87s
+```
+
+{% endcapture %}
+
+{{ gateway | indent }}
+The `Gateway` is now accessible using the external address `172.20.0.3:8080`.
+
+3. Add an [`HTTPRoute`](https://gateway-api.sigs.k8s.io/api-types/httproute/).
+
+   `HTTPRoute` resources contain a set of matching criteria for HTTP requests and upstream `Services` to route those requests to.
+
+   ```sh
+   echo "apiVersion: gateway.networking.k8s.io/v1beta1
+   kind: HTTPRoute
+   metadata:
+     name: echo
+     namespace: kuma-demo
+   spec:
+     parentRefs:
+     - group: gateway.networking.k8s.io
+       kind: Gateway
+       name: kuma
+       namespace: kuma-demo
+     rules:
+     - backendRefs:
+       - kind: Service
+         name: demo-app
+         port: 5000
+         weight: 1
+       matches:
+       - path:
+           type: PathPrefix
+           value: /
+   " | kubectl apply -f -
+   ```
+
+   After creating an `HTTPRoute`, accessing `/` forwards a request to the demo app:
+
+   ```sh
+   curl 172.20.0.3:8080/ -i
+   ```
+
+   ```
+   HTTP/1.1 200 OK
+   x-powered-by: Express
+   accept-ranges: bytes
+   cache-control: public, max-age=0
+   last-modified: Tue, 20 Oct 2020 17:16:41 GMT
+   etag: W/"2b91-175470350a8"
+   content-type: text/html; charset=UTF-8
+   content-length: 11153
+   date: Fri, 18 Mar 2022 11:33:29 GMT
+   x-envoy-upstream-service-time: 2
+   server: Kuma Gateway
+
+   <html>
+   <head>
+   ...
+   ```
+
+### TLS termination
+
+Gateway API supports TLS termination by using standard `kubernetes.io/tls` Secrets.
+
+Here is an example
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret-tls
+  namespace: kuma-demo
+type: kubernetes.io/tls
+data:
+  tls.crt: "MIIEOzCCAyO..." # redacted
+  tls.key: "MIIEowIBAAKC..." # redacted
+```
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: kuma
+  namespace: kuma-demo
+spec:
+  gatewayClassName: kuma
+  listeners:
+    - name: proxy
+      port: 8080
+      hostname: test.kuma.io
+      protocol: HTTPS
+      tls:
+        certificateRefs:
+          - name: secret-tls
+```
+
+Under the hood, {{site.mesh_product_name}} CP copies the `Secret` to `{{site.mesh_namespace}}` namespace and converts it to {% if_version lte:2.1.x %}[{{site.mesh_product_name}} secret](/docs/{{ page.version }}/security/secrets){% endif_version %}{% if_version gte:2.2.x %}[{{site.mesh_product_name}} secret](/docs/{{ page.version }}/production/secure-deployment/secrets/){% endif_version %}.
```

Comment:
```suggestion
Under the hood, {{site.mesh_product_name}} CP copies the `Secret` to `{{site.mesh_namespace}}` namespace and converts it to {% if_version lte:2.1.x inline:true %}[{{site.mesh_product_name}} secret](/docs/{{ page.version }}/security/secrets){% endif_version %}{% if_version gte:2.2.x inline:true %}[{{site.mesh_product_name}} secret](/docs/{{ page.version }}/production/secure-deployment/secrets/){% endif_version %}.
```

Reply:
why do we need inline in these? they are rendered correctly 

<img width="927" alt="image" src="https://github.com/kumahq/kuma-website/assets/2753650/8a5c5375-384a-4e56-8d13-a566006329a3">

I thought inline is mostly for tables or things that do not render correctly without it.